torch/csrc/api/src/nn/init.cpp

```diff
@@ -35,16 +35,68 @@ struct Fan {
   int64_t out;
 };
 
+#define COMPUTE_NONLINEARITY_ENUM(name) /* NOLINT(cppcoreguidelines-macro-usage) */ \
+case Nonlinearity::name: \
+  TORCH_WARN( \
```

Comment:
`TORCH_WARN_ONCE`?

Reply:
I do want people to move to the new enum style sooner than later, because the old style will be deprecated soon :D I think it's probably okay for the warning to be a bit more annoying here, since moving to the new style is minimal effort.